test/import/FSHImporter.Alias.test.ts

```diff
@@ -31,6 +34,70 @@ describe('FSHImporter', () => {
       expect(result.aliases.get('UCUM')).toBe('http://unitsofmeasure.org');
     });
 
+    it('should report when the same alias is defined twice w/ different values in the same file', () => {
+      const input = `
+      Alias: USCoreRace = http://hl7.org/fhir/us/core/ValueSet/omb-race-category
+      Alias: USCoreRace = http://hl7.org/fhir/us/core/StructureDefinition/us-core-race
+      `;
+
+      const result = importSingleText(input);
+      expect(result.aliases.size).toBe(1);
+      expect(result.aliases.get('USCoreRace')).toBe(
+        'http://hl7.org/fhir/us/core/ValueSet/omb-race-category'
+      );
+      expect(loggerSpy.getLastMessage()).toMatch(
+        /Alias USCoreRace cannot be redefined to http:\/\/hl7.org\/fhir\/us\/core\/StructureDefinition\/us-core-race; it is already defined as http:\/\/hl7.org\/fhir\/us\/core\/ValueSet\/omb-race-category\..*Line: 3\D/s
+      );
+    });
+
+    it('should report when the same alias is defined twice w/ different values in different files', () => {
+      const input = `
+      Alias: USCoreRace = http://hl7.org/fhir/us/core/ValueSet/omb-race-category
+      `;
+      const input2 = `
+      Alias: USCoreRace = http://hl7.org/fhir/us/core/StructureDefinition/us-core-race
+      `;
+
+      const results = importText([
+        new RawFSH(input, 'Alias1.fsh'),
+        new RawFSH(input2, 'Alias2.fsh')
+      ]);
+      expect(results).toHaveLength(2);
+      expect(results[0].aliases.size).toBe(1);
+      expect(results[0].aliases.get('USCoreRace')).toBe(
+        'http://hl7.org/fhir/us/core/ValueSet/omb-race-category'
+      );
+      expect(results[1].aliases.size).toBe(0);
+      expect(loggerSpy.getLastMessage()).toMatch(
+        /Alias USCoreRace cannot be redefined to http:\/\/hl7.org\/fhir\/us\/core\/StructureDefinition\/us-core-race; it is already defined as http:\/\/hl7.org\/fhir\/us\/core\/ValueSet\/omb-race-category\..*File: Alias2\.fsh.*Line: 2\D/s
+      );
+    });
+
+    it('should not report an error when the same alias is defined twice w/ different values in different files', () => {
```

Comment:
This name is confusing, since I think you might have a copy-paste error. Could you remove the 'w/ different values' section?

Reply:
Ah.  Dang.  Good catch.